oauth2/authorization-code-flow/no-token-generation/nginx.conf

```diff
@@ -12,23 +13,26 @@ events {
 
 http {
   lua_shared_dict api_keys 10m;
+  server_names_hash_bucket_size 128;
   lua_package_path ";;$prefix/?.lua;";
   init_by_lua 'math.randomseed(ngx.time()) ; cjson = require("cjson")';
 
+  resolver 8.8.8.8 8.8.4.4;
+
   upstream backend_CHANGE_ME_API_BACKEND {
     server CHANGE_ME_API_BACKEND:CHANGE_ME_PORT max_fails=5 fail_timeout=30;
   }
 
-  upstream threescale_backend {
-    server su1.3scale.net:443; #su1.3scale.net:80; #
-  }
 
   server {
+    # Enabling the Lua code cache is strongly encouraged for production use. Here it is enabled by default for testing and development purposes
```

Comment:
Correct text:
Enabling the Lua code cache is strongly encouraged for production use. Here it is disabled by default for testing and development purposes